jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildImageTask.java

```diff
@@ -66,6 +67,14 @@ public void buildImage() throws InvalidImageReferenceException {
     // Asserts required @Input parameters are not null.
     Preconditions.checkNotNull(jibExtension);
 
+    String targetImage = jibExtension.getTargetImage();
+    if (Strings.isNullOrEmpty(targetImage)) {
+      throw new GradleException(
+          "Missing target image parameter. Add a 'to.image' configuration parameter to your "
+              + "build.gradle or set the parameter via commandline (e.g. 'gradle jib --image "
+              + "your.image/name').");
```

Comment:
We should probably just have this be like `<your image name>` without the symbols, which might be confusing.